pageserver/src/lib.rs

```diff
@@ -120,9 +121,39 @@ impl<T> TenantTimelineValues<T> {
     }
 }
 
-/// A suffix to be used during file sync from the remote storage,
-/// to ensure that we do not leave corrupted files that pretend to be layers.
-const TEMP_FILE_SUFFIX: &str = "___temp";
+/// The name of the metadata file pageserver creates per timeline.
+/// Full path: `tenants/<tenant_id>/timelines/<timeline_id>/metadata`.
+pub const METADATA_FILE_NAME: &str = "metadata";
```

Comment:
Thinking about tenant.rs/timeline.rs for corresponding consts. Currently these files are overloaded, but probably it is a good fit for theoretical `tenant/mod.rs`/`timeline/mod.rs`

Reply:
Perhaps. On the other hand, it's nice to have all the filename-related constants in one place. Some of these are per-timeline, some are per-tenant, and the temp-file suffix applies everywhere.